setup.py

```diff
@@ -56,7 +56,9 @@
     author_email="admin@apeworx.io",
     url="https://github.com/ApeWorX/ape-avalanche",
     include_package_data=True,
-    install_requires=[],
+    install_requires=[
+        "eth-ape>=0.6.0,<0.7.0",
```

Comment:
To encourage using a less buggy version, we could change 0.6.0 to 0.6.5

Reply:
@unparalleled-js sounds good to me